scanpy/external/tl/_wishbone.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+
+
+def wishbone(
+    adata: AnnData,
+    start_cell: str,
+    branch: bool = True,
+    k: int = 15,
+    components_list: list = [1, 2, 3],
```

Comment:
```suggestion
    components: Iterable[int] = (1, 2, 3),
```

and later do

```py
components = list(components)
```

That way people can just pass an ndarray or tuple or whatever.